nbs/api/14_quarto.ipynb

```diff
@@ -11,6 +11,14 @@
     "- order: 14"
```

Comment:
<p>Can you remove the output here please? (Just re-run the cell when it's a comment and the output will disappear.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>